src/main/java/apoc/export/util/Neo4jAdminImportUtil.java

```diff
@@ -0,0 +1,40 @@
+package apoc.export.util;
+
+import org.neo4j.values.storable.DurationValue;
+import org.neo4j.values.storable.PointValue;
+
+import java.time.*;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Neo4jAdminImportUtil {
+
+    private static Map<Class<?>, String> allowedMapping = new HashMap(){{
```

Comment:
did you check the odd formatting for datetime and spatial that are required? see the docs.

Reply:
the whole list is taken from here:
https://neo4j.com/docs/operations-manual/current/tools/import/file-header-format/#import-tool-header-format-properties